int16[18] data
int8 size
int8 command
int16 speed
